src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -867,7 +867,7 @@ private static String processPageSourceForGodMode(String content) {
                 // now (used in comments last edited date) e.g. [Thu, 07 May 2015, 07:52:13 UTC]
                 .replaceAll(new SimpleDateFormat("EEE, dd MMM yyyy, ").format(now) + "[0-9]{2}:[0-9]{2}:[0-9]{2} UTC", "\\${comment\\.date}")
                 // now (used in opening time/closing time Grace period)
-                .replaceAll(new SimpleDateFormat("EEE, dd MMM yyyy, ").format(now) + "[0-9]{2}:[0-9]{2}", "\\${grace\\.period\\.date}")
+                .replaceAll(new SimpleDateFormat("EEE, dd MMM yyyy, ").format(now) + "[0-9]{2}:[0-9]{2} [AP]M", "\\${grace\\.period\\.date}")
```

Comment:
Sorry for being picky, but can we change the comment and the placeholder `${grace.period.date}` there? As you can see this is used in way more places than grace period dates.